Simplified.xcodeproj/project.pbxproj

```diff
@@ -5620,10 +5599,9 @@
 				A4BA1D131B43046B006F83DF /* NYPLCatalogGroupedFeed.m in Sources */,
 				11F3773319E0876F00487769 /* NYPLCatalogFacet.m in Sources */,
 				73C3CF5425C8EB6900CA8166 /* NYPLUserAccount.swift in Sources */,
-				21F238C32499155E004DC0B1 /* AdobeDRMContainer.mm in Sources */,
+        21F238C32499155E004DC0B1 /* AdobeDRMContainer.mm in Sources */,
```

Comment:
Not sure why Xcode removed this line, I added it back manually but the spacing is a bit off